lib/config/presets/internal/packages.ts

```diff
@@ -36,18 +36,28 @@ export const presets: Record<string, Preset> = {
     description: 'All Stylelint packages.',
     matchPackagePrefixes: ['stylelint'],
   },
+  standard: {
+    description: 'JavaScript Standard Style',
+    matchPackagePrefixes: ['standard'],
```

Comment:
Is there more than one package? Because this will match `standard`, `standardxyz`, `standard-abc`, etc.